RockLib.Messaging.SQS/SQSReceiverMessage.cs

```diff
@@ -75,27 +83,31 @@ private static bool TryGetSNSMessage(string messageBody, bool unpackSNS, out SNS
             {
                 try
                 {
-                    snsMessage = JsonConvert.DeserializeObject<SNSMessage>(messageBody);
-                    if (snsMessage.TopicARN != null && snsMessage.TopicARN.StartsWith("arn:"))
+                    snsMessage = JsonConvert.DeserializeObject<SNSMessage>(messageBody)!;
+                    if (snsMessage.TopicARN is not null && snsMessage.TopicARN.StartsWith("arn:", StringComparison.InvariantCultureIgnoreCase))
+                    {
                         return true;
+                    }
                 }
+#pragma warning disable CA1031 // Do not catch general exception types
                 catch { }
+#pragma warning restore CA1031 // Do not catch general exception types
             }
 
-            snsMessage = null;
+            snsMessage = null!;
```

Comment:
If `snsMessage` can be null, should the parameter be `SNSMessage?`? Or use the `[MaybeNullWhen(false)]` attribute.